docs/content/preview/troubleshoot/nodes/check-processes.md

```diff
@@ -10,33 +10,33 @@ menu:
 type: docs
 ---
 
-## 1. Are the YugabyteDB servers running?
+To troubleshoot server issues, you should perform a number of checks.
 
-Connect to the local node where YugabyteDB is running.
+## Are the YugabyteDB servers running?
 
-On the local setup, this is your local machine (or a Docker instance running on your local machine). On a multi-node cluster, you may need to `ssh` into the machines where the YugabyteDB nodes are running.
+To verify whether or not the servers are running, you need to connect to the local node where YugabyteDB is running. On a local setup, this would be your local machine (or a Docker instance running on your local machine). On a multi-node cluster, you may need to `ssh` into the machines where the YugabyteDB nodes are running, as follows:
 
 ```sh
-$ ps aux | grep yb-tserver
+ps aux | grep yb-tserver
 ```
 
-If you are expecting a yb-master servers on this node, you can also do the following.
+If you are expecting a YB-Master servers on this node, execute the following command:
 
 ```sh
 $ ps aux | grep yb-master
 ```
 
-If the servers are not running you can start them with:
+If the servers are not running, you can start them by using the following:
 
 - `yb-ctl` when using a local cluster.
 
 - `bin/yb-tserver` and `bin/yb-master` servers when using a multi-node cluster.
 
-Once the servers are running, if they are not accessible from your client machine this may be a network issue (see below).
+Once the servers are running, yet they are not accessible from your client machine, it may indicate a network issue.
 
-## 2. Are the yb-master and yb-tserver endpoints accessible?
+## Are the YB-Master and YB-TServer endpoints accessible?
 
-Generally, the endpoints are:
+Typically, the endpoints are defined as follows:
 
 |      Description |                       URL |
```

Comment:
no YSQL?

Reply:
Don't know why YSQL is not there.
Out of scope for this ticket. I'll make a note to check later.